src/qmltypes/qmlfilter.h

```diff
@@ -139,13 +136,21 @@ class QmlFilter : public QObject
     bool allowAnimateIn() const;
     bool allowAnimateOut() const;
     Q_INVOKABLE void crop(const QRectF &rect);
-    void startUndoTracking(FilterController *controller, int row);
+
     Q_INVOKABLE void copyParameters();
     Q_INVOKABLE void pasteParameters(const QStringList &propertyNames);
 
+    // Functions for undo/redo
+    void startUndoTracking();
+    Q_INVOKABLE void startChangeParameterCommand(const QString &desc = QString());
+    void startChangeAddKeyframeCommand();
```

Comment:
For these "start" functions, I was thinking about calling them something else. Some ideas I had:
* StartUndoKeyframeMacro()
* StartUndoKeyframeSequence()
* StartUndoKeyframeCommand()

I am open to suggestions.

We would want to make sure the "end" command name matches the start name.